test/testsuites/opsrcpropagationtests.go

```diff
@@ -0,0 +1,73 @@
+package testsuites
+
+import (
+	"testing"
+
+	olm "github.com/operator-framework/operator-lifecycle-manager/pkg/api/apis/operators/v1alpha1"
+	marketplace "github.com/operator-framework/operator-marketplace/pkg/apis/operators/v1"
+	"github.com/operator-framework/operator-marketplace/test/helpers"
+	"github.com/operator-framework/operator-sdk/pkg/test"
+	apps "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+// OpSrcPropagation is a test suite that confirms that the expected kubernetets resources are
```

Comment:
Why is this `propagation`? You are testing creating here.

Reply:
Fair - I had used propagation because the test ensured that labels were propagated correctly but I think `OpSrcCreation` captures the spirit of the test better.

Reply:
Resolved in the latest commit.